_layouts/category.html

```diff
@@ -80,6 +80,10 @@
   	font-size: 0.7em;
   	color: #545454;
   }
+  .returnChevron {
```

Comment:
A tiny style suggestion: to be consistent with the rest of the styles on the page I think the class should be `.return-chevron` instead of using camel case. Looks good though!